syntax = "proto3";

package model_grpc_service;

service ModelgRPCService {

  rpc iris_model_predict (iris_model_input) returns (iris_model_output) {}

}


message iris_model_input {
  
  float sepal_length = 1;
  
  float sepal_width = 2;
  
  float petal_length = 3;
  
  float petal_width = 4;
  
}



message iris_model_output {
  
  string species = 1;
  
}
